src/oasis-indexer/api.ts

```diff
@@ -34,7 +34,8 @@ export const useGetRuntimeTransactions: typeof generated.useGetRuntimeTransactio
       ...options?.axios,
       transformResponse: [
         ...arrayify(axios.defaults.transformResponse),
-        (data: generated.RuntimeTransactionList) => {
+        (data: generated.RuntimeTransactionList, headers, status) => {
+          if (!(status && 200 <= status && status < 300)) return data
```

Comment:
Any reason why we would need to check all 2xx codes?

Reply:
Not really

Reply:
Would prefer if we would only check for code 200. If it will cause problems, we can always change.